api/src/main/scala/uk/ac/wellcome/platform/api/controllers/WorksController.scala

```diff
@@ -44,36 +43,19 @@ object IncludesValidation {
   def validateIncludes(includes: Option[String]): ValidationResult = {
     includes match {
       case Some(includesString) => {
-        val includesList = includesString.split(",").toList
-        val recognisedIncludes = getCaseClassFieldNames[WorksIncludes]
-        val badIncludes = includesList
-          .filterNot(recognisedIncludes.contains)
-        badIncludes.length match {
-          case 0 => Valid
-          case 1 => Invalid(s"includes: '${badIncludes.head}' is not a valid include")
-          case _ => Invalid(s"includes: ${badIncludes.mkString("'", "', '", "'")} are not valid includes")
+        WorksIncludes.apply(includesString) match {
```

Comment:
This is a slight misuse of `apply` - generally the apply method is intended to be used transparently, e.g.

See: https://twitter.github.io/scala_school/basics2.html#apply

For example:

```scala
case class Foo(bar: String, bat: String)

// This is a companion object (see: http://daily-scala.blogspot.co.uk/2009/09/companion-object.html)
object Foo {
  def apply(baz: String): Foo =  {
     Foo(baz, baz) 
  }
}